packages/widgets/wage-view/src/main.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types'
+import {appFactory, reports} from 'tocco-app-extensions'
+import {env, appContext} from 'tocco-util'
+
+import WageView from './components/WageView'
+
+const packageName = 'wage-view'
+
+const initApp = (id, input, events, publicPath) => {
+  env.setInputEnvs(input)
+  if (input.nullBusinessUnit) {
+    env.setBusinessUnit(env.NULL_BUSINESS_UNIT)
```

Comment:
We should align those input params. Now we have different versions.  The `docs-browser` only has `businessUnit` and the `entity-browser` has the same as here. And I think `runInBusinessUnit` is the same as `businessUnit`. Maybe we can defined how we want to name it and adjust the `setInputEnv` accordingly and create a follow up ticket to change the "wrong" one

Reply:
I already create a new issue https://toccoag.atlassian.net/browse/TOCDEV-5389